apps/dashboard/app/controllers/batch_connect/session_contexts_controller.rb

```diff
@@ -2,8 +2,6 @@
 class BatchConnect::SessionContextsController < ApplicationController
   include BatchConnectConcern
 
-  include ActiveModel::Serializers::JSON
```

Comment:
Currently the template will store like this:
<img width="723" alt="image" src="https://github.com/OSC/ondemand/assets/43995067/163f0d8c-7789-4e98-9156-eef7eba1e34e">
which will definitely fail:
<img width="552" alt="image" src="https://github.com/OSC/ondemand/assets/43995067/393eb1bb-d773-4a5d-91d0-2c818dd32f3b">
I think it's better to store it as formatted JSON so that user can have the flexibility to manually edit that file.